client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -51,6 +55,31 @@ const MailboxItemIcon = ( { mailbox } ) => {
 	return null;
 };
 
+const getProvider = ( { mailbox } ) => {
+	if ( isTitanMailAccount( mailbox ) ) {
+		return 'titan';
+	}
+
+	if ( isGoogleEmailAccount( mailbox ) ) {
+		return 'google';
+	}
+
+	if ( isEmailForwardAccount( mailbox ) ) {
+		return 'forward';
+	}
+
+	return null;
+};
+
+const trackEvent = ( { mailbox, app, context } ) => {
```

Comment:
Maybe call this `trackAppLaunchEvent` or something more explicit and/or descriptive?

Reply:
Done :)